src/components/ReactGoogleMaps/ReactGoogleMaps.js

```diff
@@ -181,39 +181,19 @@ export const ReactGoogleMaps = ({ google }) => {
   }, [allResources.length, dispatch]);
 
   useEffect(() => {
-    const setDefaultLocation = () => {
-      setCurrentLat(parseFloat(CITY_HALL_COORDINATES.latitude));
-      setCurrentLon(parseFloat(CITY_HALL_COORDINATES.longitude));
+    const fetchCoordinates = async () => {
+      try {
+        const position = await getCoordinates();
+        setCurrentLat(position.coords.latitude);
+        setCurrentLon(position.coords.longitude);
+      } catch (error) {
+        console.error("Error obtaining geolocation. Default coordinates will be used.");
+      }
     };
-    getCoordinates()
-      .then(position => {
-        if (
-          Number.isNaN(position.coords.latitude) ||
-          Number.isNaN(position.coords.longitude)
-        ) {
-          setDefaultLocation();
-        } else {
-          dispatch(
```

Comment:
I noticed we removed these setters. are they not necessary anymore?

Reply:
Correct